features/zoeken/met-adresseerbaar-object-identificatie/overzicht.feature

```diff
@@ -0,0 +1,76 @@
+# language: nl
+Functionaliteit: zoek personen met de adresseerbaar object identificatie van een adres
+
+  Als consumer
+  wil ik personen vinden door het opgeven van de adresseerbaar object identificatie van het adres waar zij staan ingeschreven
+
+  Regel: personen worden gevonden die staan ingeschreven op het adres waarmee wordt gezocht 
+
+    Scenario: één persoon staat ingeschreven op het adres waarmee wordt gezocht
+      Gegeven adres 'A1'
+        | gemeentecode (92.10) | identificatiecode verblijfplaats (11.80) |
```

Comment:
Is het noodzakelijk om gemeentecode op te geven? Die is niet relevant voor het tonen van de functionaliteit.

Zouden we hier niet in de stap definitie een defaultwaarde kunnen vullen?

Reply:
Dit is noodzakelijk om een insert te kunnen doen in lo3_adres. Gemeentecode is required

Reply:
default 0518

Reply:
Er is een nieuwe stap definitie toegevoegd waarmee wordt aangegeven in welke gemeente het adres staat. Met de gemeentenaam wordt de gemeentecode opgezocht in lo3_gemeente en wordt op basis hiervan een adresseerbaarobjectidentificatie en nummeraanduiding identificatie gegenereerd zodat dit gegeven stap kan worden hergebruikt in zoek op adresseerbaar object/nummeraanduiding identificatie scenarios